### DISCUSSION TITLE
What are you using ODict for?

### DISCUSSION THREAD

**Discussion by Nickersoft** [TheOpenDictionary/odict#554] [General]
Hey folks!

I know the community here is still relatively small, but lately there's seem to be a spike in interest in The Open Dictionary and I would love to hear more your use cases! 

Some questions to think about:

- How are you using ODict today?
- What's the most annoying thing about ODict?
- What's your favorite thing about ODict?
- Where would you like to see the project go?

Lets make ODict the world's best file format for lexical data 🚀

**Comment by ghost** [comment]
I am a bit disappointed that odict is not in a Thesaurus format, so I cannot import it into my input Keyboard.

**Reply by Nickersoft** [reply]
Hey @jujubeja! Could you elaborate a bit more?

**Reply by ghost** [reply]
> Hey @jujubeja! Could you elaborate a bit more?

hey bro 
Can it be import into the keyboard ⌨ ?
ODict is an open alternative format to [Fcitx .dict/IBus .db/Rime .yaml etc similar formats ?
](https://keyboard.futo.org/dictionaries)
If yes, please add `support import keyboard ⌨` in README.md 

I know that this requires the keyboard to support the ODict format, which is very significant

**Reply by ghost** [reply]
> hey bro Can it be import into the keyboard ⌨ ? ODict is an open alternative format to [Fcitx .dict/IBus .db/Rime .yaml etc similar formats ? ](https://keyboard.futo.org/dictionaries) If yes, please add `support import keyboard ⌨` in README.md

I looked at your link. He's `.dict`. format
https://codeberg.org/Helium314/aosp-dictionaries
Most modern keyboard Dictionary are in `.dict`
